crates/matrix-sdk-base/src/store/ambiguity_map.rs

```diff
@@ -154,60 +154,76 @@ impl AmbiguityCache {
         self.changes.entry(room_id.to_owned()).or_default().insert(event_id, change);
     }
 
-    async fn get(
-        &mut self,
+    async fn get_old_display_name(
```

Comment:
And doc comment here too please!

Reply:
f08e545c9923a6c84a1f3376a49bace9ab8704b3